src/commands/new.ts

```diff
@@ -12,84 +12,291 @@ import {
   stopSpinner,
   clearSpinners,
 } from "../tools/pretty"
+import type { ValidationsExports } from "../tools/validations"
```

Comment:
Nice thing about `import type` is that Typescript can remove it from the bundle, so we don't need to worry about these type definition imports slowing down start up time